<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefreshSetup</class>
 <widget class="QDialog" name="RefreshSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обновление данных</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="btnOpenBrowser">
     <property name="text">
      <string>Список загружаемых скважин</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="mDeleteMissingCheckBox">
     <property name="text">
      <string>Удалять отсутствующие в проекте</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="mKoordsCheckBox">
     <property name="text">
      <string>Обновить координаты</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mAddMissingCheckBox">
     <property name="text">
      <string>Добавлять отсутствующие в слое</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="mDataCheckBox">
     <property name="text">
      <string>Обновить данные</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mSelectedCheckBox">
     <property name="text">
      <string>Только выбранные</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>268</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RefreshSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RefreshSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenBrowser</sender>
   <signal>clicked()</signal>
   <receiver>RefreshSetup</receiver>
   <slot>on_button_OpenBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_OpenBrowser()</slot>
 </slots>
</ui>
